### PR TITLE
Us003 view capability role belongs to

### PR DESCRIPTION
**Pull request by DarraghHaughey-Kainos**

### CODE REVIEW

File: service/JobRoleService.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    async getJobRoles(): Promise<JobRole[]> {
+
+        try {
+            const response = await axios.get(this.API_URL+'/job-roles');
+            return response.data;
+        } catch (e) {
+            if (e.response.status == 500)
+                throw new Error('Could not get Job Roles');
+        }
+        return null as any;
```
Comment (glenn-hamilton) [inline service/JobRoleService.ts]: 😢

File: views/_includes/layouts/base.html [hunk 2]
```diff
@@ -20,18 +20,16 @@
                     </a>
                     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
                         <div class="navbar-nav">
-                            <a class="nav-item nav-link active" href="/">Home<span class="sr-only">(current)</span></a>
+                            <a class="nav-item nav-link active" href="/job-roles">Job Roles</a>
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: Why remove the `Home` link?

File: views/job-roles.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                    <th scope="col">Status</th> 
+                    <th scope="col">Capability</th>                    
+                </tr>
+            </thead>
+            <tbody>
+                {% for job in jobRoles %}
+                <tr>
+                    <td>{{ job.jobRoleTitle }}</td>
+                    <td>Available</td>     
+                    <td>{{ job.capability_name }}</td>
```
Comment (glenn-hamilton) [inline views/job-roles.html]: This seems to not render any capability names

File: service/JobRoleService.ts (hunk 1 above)
Comment (DarraghHaughey-Kainos) [reply service/JobRoleService.ts]: i know :(

File: views/_includes/layouts/base.html (hunk 2 above)
Comment (DarraghHaughey-Kainos) [reply views/_includes/layouts/base.html]: It's in the logo

File: views/job-roles.html (hunk 3 above)
Comment (DarraghHaughey-Kainos) [reply views/job-roles.html]: It's working for me?

**Review by glup100 — APPROVED** [review]
Looks good

**Review by glenn-hamilton — APPROVED** [review]

**Pull request merged**